src/README.txt

```diff
@@ -0,0 +1,10 @@
+# Local development
```

Comment:
Using `.txt` suffix to avoid being used by build process when generating docs.
Guide on how to run & deploy docs.